### ISSUE TITLE
Autotune acting up?

### ISSUE THREAD

**Issue by kindebin** [cbuchner1/CudaMiner#46]
I'm running a laptop with Quadro 3000M card (Fermi GF110GLM architecture, 450 MHz core speed and 240 shader processing units) . Installed the latest CudaMiner from 2013-12-10 (hadn't used it before) and at first was getting about 30 khash/s. I then kept tweaking and overclocking the GPU and at one point got up to 80 khash/s.  I was just defaulting everything and letting autotune do its thing. Unfortunately I didn't take note of the launch configuration when I was getting this hashrate. For whatever reason I can't seem to get these rates anymore and I'm in the single digits. I ran in debug mode and it shows maximum warps = 484 (no idea what that means) and it chooses various different configurations (eg. F10x16) but always low hash rates. I tried doing an '-I F " but no dice. Even still I don't get any of the "yay!' acknowledgement messages as I was before so I'm not sure if it is even doing anything. I've put the GPU back to defaults and no difference. 
What am I doing wrong?

**Comment by cbuchner1** [comment]
does it help to reboot the machine? Try passing the argument -m 0, or
alternatively one of -C 1 or -C 2  (either of these arguments causes a
different method to allocate memory on the GPU)

Christian

2013/12/11 kindebin notifications@github.com

**Comment by kindebin** [comment]
Thank you for such a quick response. 
Both -C 1 and -C 2 came up with an F10x16 configuration and the same ~6-8 khash/s rate.
The -m 0 option came up as F5x16 and ran at the same speed.

If it helps the exact command I'm running (w/o name & pswd of course is):
cudaminer.exe -D -o stratum+tcp://[site]:[port] -u [worker] -p [pswd]

I'm still overclocking at the moment but I can keep tweaking that if you think that is the culprit. The card has 2GB of memory and the system (if it matters) has 4 GB of ram.

Thanks in advance.

**Comment by kindebin** [comment]
So I think I figured out a bit more and am up to about 53 khash/s. Basically I can't dynamically adjust/overclock the GPU while running cudaminer. Sometimes it works but other times the hash rate climbs to 160+ khash/s which would be great except that no blocks get accepted. I think it's erroring but even with the debug I can't really tell. There seems to be no in between where I get a better speed and get the welcome "yay!!!" Also at that speed the mining pool doesn't seem to see my worker so I can only assume that it is not working. I have reboot and keep the clock speed (actually seems to be more the memory speed) down. 
I'm using this command line which I think works best.
cudaminer.exe -D -H 2 -C 2 -l F10x16 -o stratum+tcp://[site]:[port] -u [worker] -p [pswd]

Any further advice and tweaking suggestions would be appreciated otherwise I'll assume that's the limit of this machine.

On another note I have another PC with a fairly recent model MSI board and Intel Core i5-3570k quad core 3.4 GHz and 8 GB of ram. I'm using the built in Intel HD Graphics 4000 card. Not sure if there is any program or what I can use to mine on that PC. I may have an old GeForce 8600 GT (from 2007) lying around which, while old maybe better although I'm not sure. I found a good site that can compare cards.  

http://www.game-debate.com/gpu/index.php?gid=781&gid2=102&compare=intel-hd-graphics-4000-desktop-vs-geforce-8600-gt

Opinion on both topics?

Thanks.
